core/pom.xml

```diff
@@ -153,8 +143,12 @@
 			<artifactId>opennlp-tools</artifactId>
 		</dependency>
 		<dependency>
+			<!-- TODO: NemexWekaClassificationEDATest throws "java.lang.Exception: liblinear classes not in CLASSPATH!".
+            This requires a wrapper but the one in https://github.com/bwaldvogel/liblinear-weka is outdated. -->
 			<groupId>de.bwaldvogel</groupId>
 			<artifactId>liblinear</artifactId>
+			<version>2.11</version>
```

Comment:
If Weka or whatever dependency requires liblinear was built against the a v1.x version of liblinear, it will likely not work with 2.x. The 1.7.1 version on Maven Central is one that the maintainer did on special request by us because some older NLP tools make use of it. Better check which version you really need.